### PR TITLE
Improvement/endtoend test setup

### LINKED ISSUE #116: Create containers and scripts for end-to-end test setup
Currently, there is documentation on the end-to-end test for the admin user to perform it in order to check if the SAPS pipeline is OK (probably from a new deployment). However, immutable scripts are used for the test which must be kept in specific containers in addition to a new configuration for some project files, so it is necessary to create scripts to facilitate the performance of this test.

### PR DESCRIPTION
**Pull request by thiagoyeds**
Fixed #116

### CODE REVIEW

File: src/main/java/org/fogbowcloud/saps/engine/utils/ExecutionScriptTagUtil.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 	
 	protected static final String ERROR_MSG__TYPE_NOT_FOUND = "Execution Script Tag type not found.";
 	protected static String DEFAULT_SCRIPT_TAG_JSON_PATH = "resources/execution_script_tags.json";
+	protected static String DEFAULT_ENDTOEND_SCRIPT_TAG_JSON_PATH = "resources/execution_endtoend_script_tags.json";
```
Comment (thiagomanel) [inline src/main/java/org/fogbowcloud/saps/engine/utils/ExecutionScriptTagUtil.java:18]: this is something we definitely don't want to add to our code. I'm fine to accept this PR if you add a feature to have a CLI or API to submit scripts

**Review by thiagomanel — CHANGES_REQUESTED** [review]
thanks @thiagoyeds 
I'm ok to accept this PR but you need first to create an issue as requested on the comment I've submitted

**Review by raonismaneoto — APPROVED** [review]

**Comment by thiagoyeds** [comment]
Hello @thiagomanel,
I created the requested issue in #154, can you take a look and check if everything is ok?

**Comment by thiagomanel** [comment]
> Hello @thiagomanel,
> I created the requested issue in #154, can you take a look and check if everything is ok?

Thanks. Now, I'm ok to aprove the PR

**Review by thiagomanel — APPROVED** [review]
lgmt

**Pull request merged**